src/mail/mjml/installment_refused.mjml

```diff
@@ -0,0 +1,50 @@
+<mjml>
+  <mj-include path="./partial/header.mjml" />
+  <mj-body mj-class="bg--blue-100">
+    <mj-wrapper css-class="wrapper" padding="20px 40px 40px 40px">
+      <mj-section>
+        <mj-column>
+          <mj-image src="{% base64_static 'joanie/images/logo_fun.png' %}" width="200px" align="left" alt="{%trans 'Company logo' %}" />
+        </mj-column>
+      </mj-section>
+      <mj-section mj-class="bg--blue-100" border-radius="6px 6px 0 0" padding="30px 50px 10px 50px">
+        <mj-column>
+          <mj-text padding="0">
+            {% if fullname %}
+            {% blocktranslate with name=fullname%}
+            <p>Hello {{ name }},</p>
+            {% endblocktranslate %}
+            {% else %}
+            {% trans "Hello," %}
+            {% endif %}<br />
+          </mj-text>
+        </mj-column>
+      </mj-section>
+      <mj-section mj-class="bg--blue-100" border-radius="6px 6px 0 0" padding="0 50px 20px 50px">
+        <mj-column>
+          <mj-text padding="0">
+            {% blocktranslate with amount=amount|format_currency_with_symbol nth_installment=nth_installment|ordinal title=course_title %}
+            For the course <strong>{{ title }}</strong>, the {{ nth_installment }}
+            installment debit has failed.
+            <br />
+            We have tried to debit an amount of {{ amount }} on the credit card
+            •••• •••• •••• {{ credit_card_last_four_numbers }}.
+            {% endblocktranslate %}
+          </mj-text>
+        </mj-column>
+      </mj-section>
+      <mj-include path="./partial/installment_table.mjml" />
+      <mj-section mj-class="bg--blue-100" padding="0 50px 20px 50px">
+        <mj-column>
+          <mj-text padding="0">
+            {% blocktranslate %}
+            Please correct the failed payment as soon as possible using
+            your dashboard : <a href="{{ dashboard_order_link }}">dashboard</a>
```

Comment:
```
            Please correct the failed payment as soon on your dashboard. <a href="{{ dashboard_order_link }}">Regularise now!</a>
```

Reply:
I feel it is a bit rough! :D
Maybe something like
```
Please correct the failed payment as soon as possible using
<a href="{{ dashboard_order_link }}">your dashboard</a>.
```

Reply:
It seems softer that way @kernicPanel 
WDYT @jbpenrath ?

Reply:
Yup much better !